uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -333,6 +334,31 @@ public void connect(String clientDeviceId, String componentId, String brokerId,
         throw lastException;
     }
 
+    /**
+     * Creates MQTT connection.
+     *
+     * @param clientDeviceId the id of the device (thing name) as defined by user in scenario
+     * @param componentId  the componentId of MQTT client
+     * @param brokerId the id of broker, before must be discovered or added by default
+     * @param mqttVersion the MQTT version string
+     * @throws RuntimeException throws in fail case
+     *
+     */
+    @SuppressWarnings({"PMD.UseObjectForClearerAPI", "PMD.AvoidCatchingGenericException"})
+    @And("I can not connect device {string} on {word} to {string} using mqtt {string}")
+    public void canNotConnect(String clientDeviceId, String componentId, String brokerId, String mqttVersion) {
+        try {
+            Mqtt5ConnAck connAck = connect(clientDeviceId, componentId, brokerId, mqttVersion);
+            int actualStatus = connAck.getReasonCode();
+
+            if (actualStatus == 0 || actualStatus == 1) {
+                log.error("ConAck returned 'success' reason code, but expected 'failed'");
+                throw new RuntimeException("ConAck returned 'success' reason code, but expected 'failed'");
```

Comment:
CONNACK not a CONACK

Reply:
fixed in 3c7f14eebc28e5651cb9f11322d165c14f0a7067